backend/pkg/cluster/fake.go

```diff
@@ -0,0 +1,58 @@
+package cluster
+
+import (
+	"context"
+	"errors"
+
+	"github.com/kyverno/kyverno/api/kyverno/v2alpha1"
+	"github.com/kyverno/kyverno/pkg/clients/dclient"
+	engineapi "github.com/kyverno/kyverno/pkg/engine/api"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/client-go/kubernetes"
+	kubefake "k8s.io/client-go/kubernetes/fake"
+)
+
+type fakeCluster struct {
+	kubeClient kubernetes.Interface
+	dClient    dclient.Interface
```

Comment:
should be removed and the Get API also return an error

Reply:
done